### ISSUE TITLE
Any way to use this with Webpack / import ?

### ISSUE THREAD

**Issue by Dylan190774** [steffest/BassoonTracker#76]
Hi, I love this project, so good to hear those old Amiga mod's again!

I'm trying to implement the standalone player in a little web game I'm making. 
I use Vue, which is uses Webpack5 to build the app, but I only could get it working with putting :

`    <script src="js/bassoonplayer-min.js"></script>`

in the main `index.html` and use `BassoonTracker `as a global object.
I rather would like to do this in the Webpack-way, with an `import `in the Vue file that uses the player.

Any idea's?

**Comment by steffest** [comment]
Sure, I'll whip up a demo.
In the mean time you could take a look at https://github.com/webpack/docs/wiki/shimming-modules

To be frank: while I see the added value of webpack (and npm in general), I tried to stay clear of it in Bassoontracker. Webpack is like a sledgehammer approach, Bassoontracker is more like a surgical scalpel.

**Comment by Dylan190774** [comment]
I agree, but with many frameworks like Quasar, Webpack is the default build-method. Quasar will change to Vite soon, by the way. Don't know if that makes a difference :)

**Comment by steffest** [comment]
Closing this as we're using Vite now to (optionally) bundle everything.
